### PR TITLE
Replace ceph-jewel with ceph-nautilus

### PR DESCRIPTION
**Pull request by hoerup**
Don't refer to a ceph version that has been EOL for a couple of years

### CODE REVIEW

**Comment by olivierlambert** [comment]
Do we have Ceph Nautilus available in XCP-ng? Is it changing anything in regard of the rest of the doc or the way it's working?

**Comment by hoerup** [comment]
yes the storage-sig nautilus release is available on my XCP 8.1 so I assume its available on 8.2 as well
For the rest the procedure is unchanged

on that note, maybe the docs should mention that cephfs and gluster requires 8.2 ?

**Comment by stormi** [comment]
> yes the storage-sig nautilus release is available on my XCP 8.1 so I assume its available on 8.2 as well
> For the rest the procedure is unchanged
> 
> on that note, maybe the docs should mention that cephfs and gluster requires 8.2 ?

Indeed. Feel free to update your PR or create a new one to add this information.

File: docs/storage.md [hunk 1]
```diff
@@ -191,6 +199,7 @@ Now you can create the SR where `server` is your mon ip.
 :::tip
 * For `serverpath` it would be good idea to use an empty folder from the CephFS instead of `/`.
 * You may specify `serverport` option if you are using any other port than 6789.
+* Do not use admin keyring for production, but make a seperate key with only necessary privileges https://docs.ceph.com/en/latest/rados/operations/user-management/
```
Comment (stormi) [inline docs/storage.md:202]: seperate => separate (same below)

File: docs/storage.md [hunk 2]
```diff
@@ -151,6 +151,10 @@ xe sr-create content-type=user shared=true type=lvmohba name-label=MyHBAStorage
 
 ### Glusterfs
 
+:::tip
+Requires XCP-ng 8.2
```
Comment (stormi) [inline docs/storage.md:155]: "Available since: XCP-ng 8.2" would be a more future-proof phrasing.

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
The PR now has conflicts with master. I'll merge the changes in my other PR that brings many other modifications to the file.

**Comment by stormi** [comment]
Included in #47

**Pull request closed without merging**